packages/components/src/molecules/TitleBar/types.ts

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 
 export type TitleBarType = {
-  title: string;
+  title: React.ReactNode;
```

Comment:
Is there a use case for using something else instead of a string? 🤔 if it's only a matter of styling, we could add a prop for the TitleBar, something like `titleStyle` and just use it in there.

Currently the title is already styled, so just by inserting your own component I'm not sure would be of much help...